### ISSUE TITLE
v1alpha2 pytorch API should try to be consistent with TFJob

### ISSUE THREAD

**Issue by jlewi** [kubeflow/pytorch-operator#49]
TFJob operator v1alpha2 made a bunch of API changes e.g

- Use dictionary for replicas not list
- Introduced CleanPodPolicy

We should create a v1alpha2 API for PyTorch that strives to be as consistent with TFJob as possible.

**Comment by jlewi** [comment]
@johnugeorge How's this coming? Looks like the PR kubeflow/tf-operator#786 creates a v1alpha2 API for PyTorch that is consistent with TFJob v1alpha2.

**Comment by johnugeorge** [comment]
Yes. You are right. The confusion is only wrt to location where to keep the code.  I tested v1 alpha2 code with a basic example.
